Cargo.toml

```diff
@@ -1,17 +1,21 @@
 [package]
 name = "libftdi1-sys"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Why is the version bumped to `0.2.0` instead of `0.1.1`? Are there known incompatibilities?

Reply:
I think it's technically a breaking change because the `pkg-config` functionality for Windows was completely [gated](https://github.com/tanriol/libftdi1-sys/pull/1/files#diff-a7b0a2dee0126cddf994326e705a91eaR1) out. But see my comment below.